src/batch-client/index.ts

```diff
@@ -512,6 +515,14 @@ export class ZimbraBatchClient {
 			return normalized;
 		});
 
+	public sendInviteReply = (requestOptions: InviteReplyInput) =>
+		this.jsonRequest({
+			name: 'SendInviteReply',
+			body: {
+				...denormalize(InviteReply)(requestOptions)
+			}
+		}).then(res => normalize(CalendarItemHitInfo)(res));
```

Comment:
Since `normalize` returns a function, you can avoid the extra function closure here by just using the return value from `normalize`:

```js
}).then(normalize(CalendarItemHitInfo));
```